activerecord/test/cases/secure_token_test.rb

```diff
@@ -22,4 +22,11 @@ def test_regenerating_the_secure_token
     assert_not_equal @user.token, old_token
     assert_not_equal @user.auth_token, old_auth_token
   end
+
+  def test_token_value_not_overwritten_when_present
```

Comment:
nope - `@user` is a new instance of model `User` as you can see in setup: https://github.com/morgoth/rails/blob/do-not-overwrite-value-of-secret-token-when-present/activerecord/test/cases/secure_token_test.rb#L6